index.html

```diff
@@ -125,6 +125,11 @@
         text-align: center;
       }
 
+      .row {
+        margin-left: 0;
+        margin-right: 0;
+      }
```

Comment:
Hm, ideally we shouldn't override any of the Bootstrap styles if we can avoid it.

Reply:
agreed. why is this needed? it sounds like maybe we're using `.row` outside of a `.container`?